numba_dpex/experimental/kernel_dispatcher.py

```diff
@@ -13,17 +13,19 @@
 from numba.core.compiler import CompileResult
 from numba.core.compiler_lock import global_compiler_lock
 from numba.core.dispatcher import Dispatcher, _FunctionCompiler
+from numba.core.target_extension import dispatcher_registry, target_registry
 from numba.core.typing.typeof import Purpose, typeof
 
 from numba_dpex import config, spirv_generator
-from numba_dpex.core.descriptor import dpex_kernel_target
 from numba_dpex.core.exceptions import (
     ExecutionQueueInferenceError,
     UnsupportedKernelArgumentError,
 )
 from numba_dpex.core.pipelines import kernel_compiler
 from numba_dpex.core.types import DpnpNdArray
 
+from .target import dpex_exp_kernel_target
```

Comment:
do we want to avoid relative imports?

Reply:
We can, but in the past I have found relative imports to be the only way out of avoiding circular imports.

Reply:
It is okay for now then. We just need to rethink design to avoid circular imports

Reply:
I agree 👍